shared/types/User.d.ts

```diff
@@ -1,3 +1,4 @@
 export default interface User {
-  id: string;
+  mId: string;
+  alias: string;
```

Comment:
Alias should be null or string. We - users don't have to set alias

Reply:
ok

Reply:
can it be `alias? : string`